gravitee-am-repository/gravitee-am-repository-jdbc/src/main/resources/liquibase/changelogs/v4_2_27/4.2.27-change-client-column-size.yml

```diff
@@ -0,0 +1,166 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.2.27-change-client-column-size-remove-indexes
+      author: GraviteeSource Team
+      dbms: mssql
```

Comment:
**todo**: can you please add a comment explaining why we have to do it only for SQLServer ? (same for the create index)